test/controllers/jobSeeker.test.ts

```diff
@@ -4,51 +4,171 @@ import request from 'supertest'
+  seniority: 'Junior',
+  createdAt: "01/12/2020",
+  requiredSkills: jobPost1Skills()  
+}
+
+const jobPost_2 = {
+  title: 'Fullstack React- & Node.js Developer',
+  jobDescription: 'We create and operate the online shops of Klamotten. Your job is to participate in the further development of our existing shop system platform',
+  seniority: 'Senior',
+  createdAt: "01/12/2020",
+  requiredSkills: jobPost2Skills()
+}
+
+const jobPost_3 = {
+  title: 'Cloud Architect',
+  jobDescription: 'Create cloud solutions',
+  seniority: 'Junior',
+  createdAt: "01/12/2020",
+  requiredSkills: jobPost3Skills()
+}
+
+const createJobPost1 = async () => {
+  await request(app).post('/employer/jobs/techeck').send(jobPost_1)
+}
+
+const getJobPost1 = async () => {
+  await request(app).get('/employer/jobs/1').send(jobPost_1)
+}
+
+const createJobPost2 = async () => {
+  await request(app).post('/employer/jobs/techeck').send(jobPost_2)
+}
+
+const getJobPost2 = async () => {
+  await request(app).get('/employer/jobs/2').send(jobPost_2)
+}
+
+const createJobPost3 = async () => {
+  await request(app).post('/employer/jobs/google').send(jobPost_3)
+}
+
+const getJobPost3 = async () => {
+  await request(app).get('/employer/jobs/3').send(jobPost_3)
+}
+const jobPosts = [jobPost_1, jobPost_2, jobPost_3]
+
+const employer_1 = {
   info: {
-    firstName: 'duy',
-    lastName: 'nguyen',
-    contact: 1234,
-    relocate: true,
-    seniority: 'junior',
-    startingDate: '10/12/2020',
+    companyName: 'techeck',
+    companyInfo: 'techeck-home',
+    address: 'techeck-address',
   },
   credential: {
-    email: 'abc@gmail.com',
+    email: 'teckeck@gmail.com',
+    password: 'password',
+  },
+  jobPosts: [
+     getJobPost1(),
+     getJobPost2()
+  ]
+}
+
+const employer_2 = {
+  info: {
+    companyName: 'google',
+    companyInfo: 'google-home',
+    address: 'google-address',
+  },
+  credential: {
+    email: 'google1@gmail.com',
     password: 'password',
   },
+  jobPosts: [
+    getJobPost3()
+  ]
 }
+// Ok
 const loginInput = {
   email: jobSeeker.credential.email,
   password: jobSeeker.credential.password,
 }
 
+// This doesn't work
 const createJobSeeker = async () =>
-  await request(app).post('/jobSeeker/create').send(form)
+  await request(app).post('/jobSeeker/create').send(jobSeeker)
 
 const logInJobSeeker = async () =>
   await request(app).post('/jobSeeker/login/local').send(loginInput)
 
-describe('jobSeeker controller', () => {
+const registerEmployer = async () => {
+  await request(app).post('/employer').send(employer_1)
+  await request(app).post('/employer').send(employer_2)
+}
+
+const match = async (): Promise<any> => {
+  await request(app).post('/employer/jobs/match').send(jobPosts)
+  await request(app).post('/jobSeeker/match').send(jobSeeker)
+}
+
+describe('user controller', () => {
```

Comment:
I think in this file there were conflicts, but were resolved wrongly. Perhaps reset this file back to the current state as on feature-controller branch, and bring the match test to a separate file

Reply:
Ok.